src/main/java/com/ays/admin_user/model/entity/AdminUserEntity.java

```diff
@@ -92,8 +92,6 @@ public Map<String, Object> getClaims() {
 
         claims.put(AysTokenClaims.USER_ID.getValue(), this.id);
         claims.put(AysTokenClaims.USERNAME.getValue(), this.username);
-        claims.put(AysTokenClaims.USER_FIRST_NAME.getValue(), this.firstName);
-        claims.put(AysTokenClaims.USER_LAST_NAME.getValue(), this.lastName);
```

Comment:
Bu iki alanı neden buradan kaldırdık?

Reply:
Token bilgisinde Admin için bu bilgilere gerek var mı? Sadece id kullanıyoruz.

Reply:
@Rapter1990 
> Token bilgisinde Admin için bu bilgilere gerek var mı? Sadece id kullanıyoruz.

Token içerisindeki bilgiler sadece BE tarafından kullanılmıyor, aynı zamanda FE tarafında da kullanılıyor

Reply:
Durum User içinde USER_FIRST_NAME ve USER_LAST_NAME kaldıramayız. Kaldıracaksak FE AuthUser'da bunları optional olarak tanımlanması lazım. Değer gelebilirde gelmeye bilirde şeklinde.

Reply:
> Durum User içinde USER_FIRST_NAME ve USER_LAST_NAME kaldıramayız. Kaldıracaksak FE AuthUser'da bunları optional olarak tanımlanması lazım. Değer gelebilirde gelmeye bilirde şeklinde.

Mobil tarafta bu bilgilere ihtiyacımız olmadığı için kaldırmayı düşündük, Kurum tarafında kullanıyoruz.